### PR TITLE
Add typescript lsp

### PR DESCRIPTION
**Pull request by TornaxO7**
Adds the typescript-language-server

### CODE REVIEW

**Comment by TornaxO7** [comment]
I don't understand why it fails.

**Comment by jooooscha** [comment]
The error seems to be a fault with the github action. I currently do not have the time or knowledge to debug this. Please ignore for now :D

Could you expand test test as in #67? Thanks :)

File: plugins/nvim-lsp/options/servers.nix [hunk 1]
```diff
@@ -95,6 +95,10 @@ let
     texlab = {
       languages = "latex";
     };
+    typescript-language-server = {
+      languages = "typescript";
+      packages = [ nodePackages.typescript-language-server ];
```
Comment (jooooscha) [inline plugins/nvim-lsp/options/servers.nix:100]: The `serverName` should be tsserver

File: tests/plugins/lsp.nix
Comment (jooooscha) [inline tests/plugins/lsp.nix:1]: As with your other PR, you need to add the typescript-language-server to the servers list and add the missing output to the assertDiff function

**Review by jooooscha — CHANGES_REQUESTED** [review]

**Comment by jooooscha** [comment]
Implemented in b8d4b01aa7d6f500034a9bd79811b6053dc6771c

**Pull request closed without merging**